src/Compilers/Core/Portable/MetadataReference/ModuleMetadata.cs

```diff
@@ -48,10 +48,21 @@ private ModuleMetadata(PEReader peReader, IDisposable? owner, bool disposeOwner)
             _disposeOwner = disposeOwner;
         }
 
-        private ModuleMetadata(IntPtr metadata, int size, bool includeEmbeddedInteropTypes, bool ignoreAssemblyRefs)
+        private ModuleMetadata(
+            IntPtr metadata,
+            int size,
+            bool includeEmbeddedInteropTypes,
+            bool ignoreAssemblyRefs,
+            IDisposable? owner,
```

Comment:
>IDisposable? owner, [](http://example.com/codeflow?start=12&length=19)

Nit: move owner and disposeOwner parameters before the other bool parameters, so that bool flags are last params

Reply:
sgtm.

Reply:
Done!